tools/test-proxy/Azure.Sdk.Tools.TestProxy/Common/SanitizerDictionary.cs

```diff
@@ -56,6 +62,606 @@ public class SanitizerDictionary
+                ),
+                // "token": "sv=...ss=...srt=...se=...sp=...sig=" request body covered by 1052
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..WEBSITE_AUTH_ENCRYPTION_KEY"),
+                    "AZSDK025"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..decryptionKey"),
+                    "AZSDK026"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("ServiceBusDlqSupplementaryAuthorization", regex: "(?:(sv|sig|se|srt|ss|sp)=)(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK027"
+                ),
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("ServiceBusSupplementaryAuthorization", regex: "(?:(sv|sig|se|srt|ss|sp)=)(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK028"
+                ),
+                // "RequestBody": "client_id=...grant_type=...client_info=...client_secret=…scope=...", covered by 1036/1037
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..access_token"),
+                    "AZSDK029"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..AccessToken"),
+                    "AZSDK030"
+                ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "(client_id=)(?<cid>[^&]+)", groupForReplace: "cid"),
+                    "AZSDK031"
+                ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "client_secret=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK032"
+                ),
+                new RegisteredSanitizer(
+                    new BodyRegexSanitizer(regex: "client_assertion=(?<secret>[^&\\\"]+)", groupForReplace: "secret"),
+                    "AZSDK033"
+                ),
+                // new BodyKeySanitizer("$..targetModelLocation"), disabled, not a secret?
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..targetResourceId"),
+                    "AZSDK034"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..urlSource"),
+                    "AZSDK035"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..azureBlobSource.containerUrl"),
+                    "AZSDK036"
+                ),
+                new RegisteredSanitizer(
+                    new BodyKeySanitizer("$..source"),
+                    "AZSDK037"
+                ),
+                // new BodyKeySanitizer("$..resourceLocation"), disabled, not a secret?
+                new RegisteredSanitizer(
+                    new HeaderRegexSanitizer("Location", value: "example.com"),
```

Comment:
Sorry I didn't underline this earlier, this value needs to a scheme e.g. `https://example.com` so it parses as a URL

Reply:
Updated. Thanks for the callout @chlowell